frontend/src/components/search/SearchSortBy.tsx

```diff
@@ -16,15 +20,32 @@ const SORT_OPTIONS: SortOption[] = [
   { label: "Close Date (Descending)", value: "closeDateDesc" },
```

Comment:
Nit: As this is a combination of `SORT BY` and `ORDER BY`, wondering if there's a more semantic name? It might be okay to just call it sort "sort." But I notice we're calling these options "sort" while the component and its other vars use "sort by." I could also see an iteration where we separate the concerns with a distinct order toggle. 

^ Feel free to ignore all that; I'm just getting thoughts out. 

However, I am curious about the labels. @crystabelrangel, you might have thoughts on this; I don't think we thought through this enough. It may be that ascending and descending are a bit of a riddle for users. Instead, might use more explicit `order` terms for each `sort`: 

- Opportunity Number ~(Ascending)~
- ~Opportunity Number (Descending)~ `// do we need both of these?`
- Opportunity Title (A–Z)
- Opportunity Title (Z–A)
- Agency (A–Z)
- Agency (Z–A)
- Posted Date (newest)
- Posted Date (oldest)
- Close Date (upcoming/new)
- Close Date (oldest)

Reply:
in meeting we noted that this would be a separate PR